src/pages/favorites/favorites.component.ts

```diff
@@ -164,6 +167,14 @@ export class FavoritesComponent {
   }
   deleteTrip(trip): void {
     _.remove(this.trips, {name: trip.name});
-    this.tripSvc.deleteTrip(trip);
+    this.faveTripSvc.deleteTrip(trip);
+  }
+  removeRoute(route): void {
+    _.remove(this.routes, {name: route.name});
+    this.faveRouteSvc.remove(route);
+  }
+  removeStop(stopId): void {
+    _.remove(this.stops, {stopId: stopId});
```

Comment:
Similar issue here as above, but this one's more subtle - `stopId` is the param that's passed to this function, but a stop object doesn't have a `stopId` property - it's called ` StopId` (note the capital `S`).  Woof.